modules/serverless-installing-cli-linux-ibm-z_rpm.adoc

```diff
@@ -0,0 +1,53 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-linux_ibm_z_rpm_{context}"]
+
+== RPM Installation
+
+For Red Hat Enterprise Linux (RHEL), you can install `kn` as an RPM if you have an active {product-title} subscription on your Red Hat account.
```

Comment:
Again, not sure about specifying RHEL like this, is this a usual thing @vikram-redhat @sjstout @bergerhoffer ?